docs/pages/database-access/guides/ha.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Database Access High Availability
+description: How to configure Teleport Database Access in HA configuration
+---
+
+# Database Access High Availability
+
+You can deploy Database Access in a highly available configuration in a couple
+of common ways: separate replicas and combined replicas. Both of those revolve
+around pointing multiple Database Services to the same database instance.
+
+## Separate Replicas
+
+With separate replicas, each Database Service agent proxying the database
+assigns it a different name. This allows you to explicitly pick the agent
+you want to connect to the database over.
+
+For example, you may have two Database Services deployed in different
+availability zones providing access to the same PostgreSQL database:
+
+```yaml
+# Database service agent #1.
+db_service:
+  enabled: "yes"
+  databases:
+  # Note the name is different than agent #2 but URI is the same.
+  - name: "postgres-us-east-1a"
+    protocol: "postgres"
+    uri: "postgres.example.com:5432"
+```
+
+```yaml
+# Database service agent #2.
+db_service:
+  enabled: "yes"
+  databases:
+  # Note the name is different than agent #1 but URI is the same.
+  - name: "postgres-us-east-1b"
+    protocol: "postgres"
+    uri: "postgres.example.com:5432"
+```
+
+With this configuration, both services will appear as two separate entries in
+`tsh db ls` output and you will have to pick one explicitly when connecting:
+
+```bash
```

Comment:
I think we're still in the process of creating dedicated code snippet sections:

1. Should automatically add the `$`.
2. Should prevent `$` from being copied (annoying to copy this on ZSH for example).
3. Will hopefully more cleanly divide "output" examples and "input" ones - at least I've requested this.

We'll likely want to revamp all of the code snippets once this done in: https://github.com/gravitational/next/issues/275

Reply:
Let me know when this change is through and I can update the snippets here.